res/values/setup.xml

```diff
@@ -4,10 +4,6 @@
     <!-- App name  and other strings-->
     <string name="app_name" formatted="false">ownCloud</string>
     <string name="account_type" formatted="false">owncloud</string>	<!-- better if was a domain name; but changing it now would require migrate accounts when the app is updated -->
-    <string name="authority" formatted="false">org.owncloud</string>
```

Comment:
The idea is good but unfortunately these four options can not be taken off from here since are used by our branding systems. Would it be possible to keep them here and read them from `build.gradle` and add .debug suffix in debug buildType?

Reply:
you meannot specify if in xml and do it in `build.gradle` ? Please confirm @davigonz

Reply:
Now I add a `dynamic_XXX` way to keep this what-ever-is `branding systems`.
It becomes now not more easy to understand